esp_debug_adapter/debug_adapter/debug_backend/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+future = ">=0.17"
+psutil = "==5.5.1"
+pygdbmi = "==0.9.0.0"
+typing = ">=3.6.6"
+xmlrunner = "==1.7.7"
+unittest-xml-reporting = "*"
```

Comment:
@an-dr I think we need to clean up `debug_backend` dependencies. From my point of view we need to leave `pygdbmi` only here.

Reply:
@brianignacio5 FYI